rca/users/signals.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+from django.db.models.signals import m2m_changed
+
+from rca.people.models import StudentIndexPage, StudentPage
+
+from .models import User
+
+logger = logging.getLogger(__name__)
+
+
+def do_after_creating_student_user(sender, **kwargs):
+
+    user = kwargs["instance"]
+
+    if user.groups.filter(name="Students").exists():
+        try:
+            student_index = StudentIndexPage.objects.first()
+        except StudentIndexPage.DoesNotExist:
+            return
+
+        student_page = StudentPage.objects.filter(
+            first_name=user.first_name, last_name=user.last_name,
```

Comment:
Is first and last name a reliable way to check, having more than one student with the same name seems quite feasible?

Reply:
Yea I'm not honestly too sure, we do have emails but the possibility of a different email between user and page I think is higher than names changing.

Reply:
Up coming in #618 I'm adding a field to 'create_student_page' so this adds an extra measure